api/src/models/student.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+const uniqueValidator = require("mongoose-unique-validator");
+/**
+ * @swagger
+ *  components:
+ *   schemas:
+ *    Student:
+ *      type: object
+ *      required:
+ *        - name
+ *        - upi
+ *      properties:
+ *        name:
+ *          type: string
+ *          description: name of the student
+ *        studentID:
```

Comment:
Should this be id instead? That's what it says in the class diagram

Reply:
what do you mean?

Reply:
oh nvm